tests/crypto/ec_tests.c

```diff
@@ -5,11 +5,13 @@
 #include <openenclave/enclave.h>
```

Comment:
It might not be a bad idea to use oe_ec_public_key_equal and oe_ec_private _key_equal to test to see if calling those kdf key API with same parameters will get the same key back

Reply:
The sealkey test should cover it, but it's pretty easy to add another test here.